.eslintrc.json

```diff
@@ -19,7 +19,7 @@
   "parserOptions": {
     "ecmaVersion": 2018,
     "sourceType": "module",
-    "project": ["./tsconfig.json"]
+    "project": ["./src/tsconfig.json"]
```

Comment:
Curious on your thoughts around this moving inside of src?

Reply:
Moving the entry tsconfig.json inside src prevents the `dist/src/...` folder problem, and having to mv/rm to work around that. Hope that helps.

Reply:
Also there might be another way of solving that problem that I'm not aware of, but it was exacerbated with the cjs/esm output folders change, so I went ahead and moved the file to fix that issue.

Reply:
Yeah, playing around with this now, and I think changing `"outDir": "dist/cjs",` in tsconfig will help.